internal/app/feed/exporter_sql.go

```diff
@@ -119,6 +119,19 @@ func (e *SQLExporter) WriteRows(feed Feed, rows interface{}) error {
 	return nil
 }
 
+// BulkUpdateRows batch updates. Returns number of rows updated or error. Works with single PKey, not with composed PKeys
+func (e *SQLExporter) BulkUpdateRows(feed Feed, primaryKeys []string, element map[string]interface{}) (int64, error) {
+	result := e.DB.
+		Table(feed.Name()).
+		Where(fmt.Sprintf("%s in ?", feed.PrimaryKey()[0]), primaryKeys).
```

Comment:
I will try this. Had some issues with composed PKs

Reply:
//NOTE: WHERE CLAUSE and why using fmt.Sprintf instead of Where(primaryKeys):
	//primaryKeys[0] -> UPDATE `inspections` SET `deleted`=true WHERE audit_47ac0dce16f94d73b5178372368af162
	//primaryKeys    -> UPDATE `inspections` SET `deleted`=true WHERE `inspections`. = "audit_47ac0dce16f94d73b5178372368af162"
	//fmt.Sprintf	 -> UPDATE `inspections` SET `deleted`=true WHERE audit_id in ("audit_47ac0dce16f94d73b5178372368af162")